transformations/stopword_removal/README.md

```diff
@@ -0,0 +1,14 @@
+# Stopword Removal
+Removes stopwords from a piece of text.
```

Comment:
Hi @juanyiloke please add your name, email and affiliation.

Reply:
done!

Reply:
thanks!